tests/distributed/test_pipeline_parallel.py

```diff
@@ -195,7 +195,7 @@ def iter_params(self, model_id: str):
     "microsoft/Phi-3-small-8k-instruct": PPTestSettings.fast(),
     "microsoft/Phi-3.5-MoE-instruct": PPTestSettings.detailed(multi_node_only=True, load_format="dummy"),  # noqa: E501
     "Qwen/Qwen-7B-Chat": PPTestSettings.fast(),
-    "Qwen/Qwen2-7B-Instruct": PPTestSettings.fast(),
+    "Qwen/Qwen2.5-1.5B-Instruct": PPTestSettings.fast(),
```

Comment:
```suggestion
    "Qwen/Qwen2.5-0.5B-Instruct": PPTestSettings.fast(),
```

Perhaps we can use 0.5B model here as well?